built-in-policies/README.md

```diff
@@ -44,6 +46,27 @@ This version is incremented according to the following rules (subject to change)
          - Example:  1.3.2-preview
       - Append "-deprecated" to the version if the policy is in a deprecated state
          - Example:  1.3.2-deprecated
-  
+ 
+## Preview and deprecated policies
+
+This section aims to explain what it means when a built-in policy has a state of ‘preview’ or ‘deprecated’.  
+
+Policies can be in preview because a property (alias) referenced in the policy definition is in preview, or the policy is newly introduced and would like additional customer feedback. A policy may get deprecated when the property (alias) becomes deprecated & not supported in the resource type's latest API version, or when there is manual migration needed by customers due to a breaking change in a resource type's latest API version. 
+
+When a policy gets deprecated or gets out of preview, there is no impact on existing assignments. Existing assignments continue to work as-is. The policy is still evaluated & enforced like normal and continues to produce compliance results.  
+
+Here are the changes that occur when a policy gets deprecated: 
+- Display name is appended with ‘[Deprecated]:’ prefix, so that customers have awareness to migrate or delete the policy.  
+- Description gets updated to provide additional information regarding the deprecation. 
+- The version number is updated with ‘-deprecated’ suffix. (see [Versioning](#versioning) above) 
+
+To deter customers from making any new assignments, deprecated definitions are hidden in the definition list view in Azure Portal. 
+
+Here are the changes that occur when a policy gets out of preview: 
+- Display name is no longer appended with ‘[Preview]:’ prefix. 
+- The version number is updated to remove the ‘-preview’ suffix. (see [Versioning](#versioning) above) 
+
+For customers who do not want such changes to impact their policy definitions, we recommend duplicating built-in policy definitions & assigning them as custom definitions. Customers can also choose to be notified of updates by [subscribing](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) to this Github repository.  
```

Comment:
NIT: could you remove the /en from the url?

Reply:
Fixed!